common/solver/gmres_kernels.hpp.inc

```diff
@@ -300,7 +299,7 @@ __device__ void calculate_residual_norm_kernel(
     const auto this_rnc =
         residual_norm_collection[iter * stride_residual_norm_collection +
                                  col_idx];
-    const auto next_rnc = -register_sin * this_rnc;
+    const auto next_rnc = -conj(register_sin) * this_rnc;
     residual_norm_collection[iter * stride_residual_norm_collection + col_idx] =
         register_cos * this_rnc;
```

Comment:
`norm_collection(iter) = cos * this_rnc + sin * next_rnc = cos * this_rnc;`
`norm_collection(iter+1) = -conj(sin) * this_rnc + cos * next_rnc = -conj(sin) * this_rnc;`
Is that correct?